scripts/import_cldr.py

```diff
@@ -693,6 +697,21 @@ def main():
         write_datafile(data_filename, data, dump_json=dump_json)
 
 
+def parse_interval_formats(data, tree):
+    # http://www.unicode.org/reports/tr35/tr35-dates.html#intervalFormats
+    interval_formats = data.setdefault("interval_formats", {})
+    for elem in tree.findall("dateTimeFormats/intervalFormats/*"):
+        if elem.tag == "intervalFormatFallback":
+            interval_formats[None] = elem.text
+        elif elem.tag == "intervalFormatItem":
+            skel_data = interval_formats.setdefault(elem.attrib["id"], {})
```

Comment:
Should we filter out draft elements? Babel currently ignores them for a lot of settings, although I haven't checked if it ignores them for all settings.